contracts/modules/TransferManager/GTM/GeneralTransferManager.sol

```diff
@@ -188,6 +190,22 @@ contract GeneralTransferManager is GeneralTransferManagerStorage, TransferManage
         return (Result.NA, bytes32(0));
     }
 
+    function _isValidModule(address _holder) internal view returns(bool) {
```

Comment:
can we change name to _isWhitelistModule

Reply:
Fixed